spec/content_metadata_spec.rb

```diff
@@ -511,6 +511,27 @@
     expect { described_class.create_content_metadata(druid: TEST_DRUID, objects: objects) }.to raise_error(RuntimeError, "File '#{junk_file}' not found")
   end
 
+  it 'generates valid content metadata for a 3d object with a single obj file and 2 other supporting files' do
```

Comment:
doesn't it have 4 files now?

Reply:
yes, will adjust name the name of the test